asmrun/arm.S

```diff
@@ -17,7 +17,7 @@
 
         .syntax unified
         .text
-#if defined(SYS_linux_eabihf) && defined(MODEL_armv6)
+#if (defined(SYS_linux_eabihf) || defined(SYS_netbsd)) && defined(MODEL_armv6)
```

Comment:
I agree and was tempted to refactor some of the symbols, but I didn't want to break things. If Linux was specified by 'SYS_linux' and we had symbols for 'eabi' and 'eabi_hf', some of my 'SYS_netbsd' tests would go away (since it would simply use the 'eabi_hf' code.)